src/app/src/components/QuizHome.js

```diff
@@ -69,6 +85,25 @@ class QuizHome extends Component {
             <QuizMedallion value={bonusPoints} isFinalScorePage={true} />
         );
 
+        const defaultOptions = {
+            loop: true,
+            autoplay: true,
+            animationData: animationData,
+            rendererSettings: {
+                preserveAspectRatio: 'xMidYMid slice',
+                clearCanvas: true,
+                progressiveLoad: true,
+            },
+        };
+
+        const FishSwimming = () => {
```

Comment:
Since this function only returns the JSX and has no other logic, we can implicitly return the JSX without the return statement. Something like:

```
const fishSwimming = () => (
    <StyledFishSwiming>
        <Lottie options={defaultOptions} />
    </StyledFishSwiming>
);
```